eclass/rakudo.eclass

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: rakudo.eclass
+# @MAINTAINER:
+# amano.kenji <amano.kenji@proton.me>
+# @BLURB: An eclass for raku modules
+
+EXPORT_FUNCTIONS src_compile src_install src_test
+
+# @ECLASS_VARIABLE: rakudo_test_deps
+# @DESCRIPTION:
+# Packages that shouldn't run tests with dev-raku/App-Prove6
+# @INTERNAL
+declare -A rakudo_test_deps
+rakudo_test_deps[dev-raku/App-Prove6]=1
+rakudo_test_deps[dev-raku/Getopt-Long]=1
+rakudo_test_deps[dev-raku/TAP]=1
+
+if [ ${rakudo_test_deps[${CATEGORY}/${PN}]} ]; then
```

Comment:
We use Bash tests were possible (so [[ ... ]], not [ .. ] ), see https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101.

Reply:
Done. All of it comes from maintaining my personal overlay.